package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -602,19 +609,50 @@ def _read_next_timestep(self):
             except IndexError:
                 raise ValueError("No coordinate information detected")
         elif not self.lammps_coordinate_convention in convention_to_col_ix:
-            raise ValueError(f"No coordinates following convention {self.lammps_coordinate_convention} found in timestep")
+            raise ValueError("No coordinates following convention"
+                    + f"{self.lammps_coordinate_convention} found in timestep")
 
         coord_cols = convention_to_col_ix[self.lammps_coordinate_convention]
 
         ids = "id" in attr_to_col_ix
+
+        # Create the data arrays for additional attributes which will be saved
+        # under ts.data
+        additional_keys = []
+        if len(attrs) > 3:
+            for attribute_key in attrs:
+                # Skip the normal columns
+                if attribute_key == "id" or \
+                        attribute_key in \
+                        self._coordtype_column_names[
```

Comment:
I think you want to test that they are not any of the coordinate types not just the coordinate convention we have chosen if that makes sense?

Reply:
Not sure what you mean by this?